client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/screen-category-list.tsx

```diff
@@ -48,31 +46,46 @@ const ScreenCategoryList = ( {
 	const categoriesInOrder = useCategoriesOrder( categories );
 	const composite = useCompositeState( { orientation: 'vertical' } );
 
-	const handleFocusOutside = ( event: Event ) => {
-		// Click outside the sidebar or action bar to close Pattern List
-		const target = event.target as HTMLElement;
-		if (
-			! (
-				target.closest( '.pattern-action-bar' ) || target.closest( '.pattern-assembler__sidebar' )
-			)
-		) {
-			setSelectedCategory( null );
-			onTogglePatternPanelList?.( false );
-		}
-	};
-
 	const trackEventCategoryClick = ( name: string ) => {
 		recordTracksEvent( PATTERN_ASSEMBLER_EVENTS.SCREEN_CATEGORY_LIST_CATEGORY_CLICK, {
 			pattern_category: name,
 		} );
 	};
 
 	useEffect( () => {
-		wrapperRef?.current?.addEventListener( 'click', handleFocusOutside );
+		const handleFocusOutside = () => {
+			setSelectedCategory( null );
+			onTogglePatternPanelList?.( false );
+		};
+
+		const handleClick = ( event: Event ) => {
+			// Click outside the sidebar or action bar to close Pattern List
+			const target = event.target as HTMLElement;
+			if (
+				! (
+					target.closest( '.pattern-action-bar' ) || target.closest( '.pattern-assembler__sidebar' )
+				)
+			) {
+				handleFocusOutside();
+			}
+		};
+
+		// Clicking the large preview iframe won't trigger the click event on the current window.
+		// Instead, the current window will blur and that iframe will be focused.
+		const handleBlur = () => {
```

Comment:
This workaround doesn't seems to work on Safari because the Pattern List doesn't close.

https://user-images.githubusercontent.com/1881481/236124190-df7223dc-05ed-48dd-86c2-ed9ca2dc3752.mov